tools/webpacks/highcharts.webpack.mjs

```diff
@@ -130,15 +133,54 @@ const webpacks = FSLib
                 )
             ];
         }
-        return webpackConfig;
+        return umdWebpack;
     });
 
 
+/**
+ * ES module bundles
+ */
+const esmWebpacks = umdWebpacks.map(umdWebpack => {
+    const esmWebpack = JSON.parse(JSON.stringify({
+        ...umdWebpack,
+        experiments: {
+            outputModule: true
+        },
+        externals: void 0,
+        externalsType: 'module',
+        output: {
+            filename: umdWebpack.output.filename,
+            libraryTarget: 'module',
+            module: true,
+            path: Path.resolve(Path.join(targetFolder, 'es-bundles'))
```

Comment:
Discussion point: Should we output the bundles to a new folder `esm`? It's cleaner, more aligned with standards and provides a clearer separation between these bundles, and building from masters inside `es-modules`. Doesn't get any cleaner than this:

  ```html
  <script type="module">
  import Highcharts from 'https://code.highcharts.com/esm/highcharts.js';
  import 'https://code.highcharts.com/esm/modules/exporting.js';
  </scripts>
  ```

```suggestion
            path: Path.resolve(Path.join(targetFolder, 'esm'))
```

Reply:
Yes, I like it. 👍